saleor/tests/fixtures.py

```diff
@@ -531,6 +531,24 @@ def customer_user(address):  # pylint: disable=W0613
     return user
 
 
+@pytest.fixture
+def customer_user_different_billing_and_shipping_address(
```

Comment:
Not sure if you really need this fixture. You can use `customer_user` and just change one of the addresses. (For your case probably no need to call `.save` )